Sources/SafeDI/DelayedInstantiation/ErasedInstantiator.swift

```diff
@@ -19,34 +19,40 @@
 // SOFTWARE.
 
 /// A SafeDI dependency designed for the deferred instantiation of a type-erased instance of a
-/// type decorated with `@Instantiable`. Instantiation is thread-safe.
```

Comment:
Given that we're explicitly determining threading here it seemed duplicative to say `Instantiation is thread-safe.`